maat-court-data-api/src/test/java/gov/uk/courtdata/billing/repository/ApplicantCclfResetRepositoryTest.java

```diff
@@ -0,0 +1,39 @@
+package gov.uk.courtdata.billing.repository;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.mockito.Mockito.*;
+
+class ApplicantCclfResetRepositoryTest {
+
+    @Test
+    void shouldExecuteStoredProcedureSuccessfully() {
+        NamedParameterJdbcTemplate mockJdbcTemplate = Mockito.mock(NamedParameterJdbcTemplate.class);
```

Comment:
Similar comment as to previous that I don't think we need to introduce `NamedParameterJdbcTemplate`.